python/setup.py

```diff
@@ -18,7 +18,7 @@
     platforms='linux',
     keywords=['kubernetes', 'docker', 'container', 'metaparticle'],
     # latest from https://pypi.python.org/pypi?%3Aaction=list_classifiers
-    classifiers = [
+    classifierssss = [
```

Comment:
This is weird, why the extra sss?

Reply:
Oh. Must be a typing issue. I'll correct it.